dev-games/libsmacker/libsmacker-1.1.1.ebuild

```diff
@@ -3,12 +3,11 @@
 
 EAPI=7
 
-inherit autotools eutils
+inherit autotools
 
 DESCRIPTION="A cross-platform C library for decoding .smk Smacker Video files."
 HOMEPAGE="https://libsmacker.sourceforge.net"
-
-SRC_URI="https://sourceforge.net/projects/libsmacker/files/libsmacker-1.1/libsmacker-1.1.1r35.tar.gz/download -> ${P}.tar.gz"
+SRC_URI="https://sourceforge.net/projects/${PN}/files/${PN}-1.1/${P}r35.tar.gz/download -> ${P}.tar.gz"
```

Comment:
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D

Don't confuse PN and P

Reply:
I did not? its exactly what ajak was asking for.

Reply:
ajak said "to use variables with versions". PN is not a variable with version. It's a variable without version, which defeats the whole point of the change.
P or PV are variables with version.

Reply:
<ajak> sf.net/projects/${PN}/files/${PN}-1.1/${P}r35.tar.gz/download

Reply:
Yes, P is good. But PN doesn't belong here.